viewer/packages/pointclouds/src/potree-three-loader/workers/assignPointsWithWasm.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { StylableObject } from '../../styling/StylableObject';
+import { assignPoints } from '../../../wasm';
+import { Cylinder } from '../../styling/shapes/Cylinder';
+import { Box } from '../../styling/shapes/Box';
+
+import { SerializedPointCloudObject } from '../../../wasm';
+import { ShapeType } from '../../styling/shapes/IShape';
+import { assertNever } from '@reveal/utilities';
+
+function createSerializedObject(obj: StylableObject): SerializedPointCloudObject {
+  switch (obj.shape.shapeType) {
+    case ShapeType.Cylinder: {
+      const cylinder = obj.shape as Cylinder;
+      return {
+        object_id: obj.objectId,
+        cylinder: {
+          center_a: cylinder.centerA,
+          center_b: cylinder.centerB,
+          radius: cylinder.radius
+        }
+      };
+    }
+    case ShapeType.Box: {
+      const box = obj.shape as Box;
+      return {
+        object_id: obj.objectId,
+        oriented_box: {
+          inv_instance_matrix: box.invMatrix.data
+        }
+      };
+    }
+    case ShapeType.Composite: {
+      throw Error('Composite types should not be sent to the parser worker, they should be decomposed first');
```

Comment:
This will at some point get hit in the future. Is it possible to formulate this with a little more user friendly wording?